component/app.jsonnet

```
@@ -6,6 +6,11 @@ local argocd = import 'lib/argocd.libjsonnet';
 
 local syn_project = argocd.Project('syn');
 local default_project = argocd.Project('default') {
+  metadata: {
```

```suggestion
  metadata+: {
```

To not delete namespace and name.

---

Oh 😅